src/colordetection.py

```diff
@@ -5,22 +5,29 @@
 # Created       : Tue, 26 Jan 2016
 # Last Modified : Sun, 31 Jan 2016
 
+# Built-ins
+from sys import exit as die
+from typing import Tuple
 
-from sys import exit as Die
+# 3rd Party
 try:
-    import sys
+    from numpy import ndarray as np_ndarray     # type: ignore
```

Comment:
Can `# type: ignore` be removed? If not, please remove leading spaces in between `np_ndarray` and `# type: ignore`